gitpod-io/gitpod
# Update product icon

## Description

Update product icon

## Related Issue(s)

Fixes https://linear.app/gitpod/issue/EXP-1083

## How to test

1. Start workspace
2. Check the correct logo is shown in the loading page

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - jp-content-leopard</li>
	<li><b>🔗 URL</b> - <a href="https://jp-content-leopard.preview.gitpod-dev.com/workspaces" target="_blank">jp-content-leopard.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - jp-content-leopard-gha.22224</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-jp-content-leopard%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-core-dev" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
